indico/web/templates/pagination.html

```diff
@@ -0,0 +1,22 @@
+{% macro render_pagination(pagination) %}
+    {% if pagination.pages > 1 %}
+        <div class="pagination">
+            {%- for page in pagination.iter_pages() %}
+                {% if page %}
+                    <span class="page-number">
+                        {% if page != pagination.page %}
+                            <a href="{{ caller(page)|trim }}" class="i-button js-pagination-element">{{ page }}</a>
+                        {% else %}
+                            <a href="#" class="i-button highlight">{{ page }}</a>
+                        {% endif %}
+                    </span>
+                {% else %}
+                    <span>…</span>
+                {% endif %}
+            {%- endfor %}
+            <span class="page-number">
+                <a href="{{ caller('all')|trim }}" class="i-button">{% trans %}Show all events{% endtrans %}</a>
```

Comment:
button for each page along with highlighted show-all button? Currently we can't use `iter_pages` for that for sure since show_all = True affects `pages` and `per_page` attributes of `Pagination` object.